### DISCUSSION TITLE
why ExplorerPatcher declared suspicious by chrome / windows ?

### DISCUSSION THREAD

**Discussion by alexisdal** [valinet/ExplorerPatcher#3015] [Q&A]
I just got a win 11 laptop. the location of the taskbar in the bottom of the screen is quite painful. SUPER irratating. I was struggle to just move it on the left part of the screen like I did many year ago on my win 10. just dragging it did nothing. To my surprise, after searching a bit, I understodd that microsoft simply disabled that feature entirely because it makes their life harder and it seems most poeple dont care anyway... 🤬🤬🤬

i stumbled upn this video => https://www.youtube.com/watch?v=CED25OhKPjI

that suggest to install a 3rd party tool to add the missing feature on window 11. and it's open source! great!

but when I try to download it, my chrome (Version 122.0.6261.113 (Official Build) (64-bit)) declares it suspicious.

![image](https://github.com/valinet/ExplorerPatcher/assets/5380181/f89ddc53-ee74-4157-b616-c7212538e370)

![image](https://github.com/valinet/ExplorerPatcher/assets/5380181/c61026c0-ccf3-48f7-aac9-0aab830dc66d)

 https://support.google.com/chrome/answer/6261569?visit_id=638461846308654859-4013223814&p=ib_download_blocked&hl=en&rd=1
 
 I suggest to somehow modify the install file because it will make people nervous. it makes me nervous.
 maybe it's about signing the software?

**Comment by pyrates999** [comment]
Chrome always does that to new files that haven't been downloaded much.  It's also because the file isn't signed.  You can safely download it.
